### ISSUE TITLE
Search results menu gets stuck

### ISSUE THREAD

**Issue by SwiftFast** [IsraelHikingMap/Site#390]
- Go to https://israelhiking.osm.org.il
- Type "התחנה המרכזית לב המפרץ" in the search box, hit enter
- When the results appear, click the first one.
- The map will pan and zoom to the proper area, but the  results box will not close, making further searches impossible.

This does not seem to happen for all POIs. I couldn't find a clear pattern. Normally, the results box should disappear.

Tested with Chrome, Firefox

**Comment by SwiftFast** [comment]
Looked into it further: Both browsers actually emit a console error when clicking "תחנה מרכזית לב המפרץ":

Chrome:

```
vendor.a85cfb9….bundle.js:1 ERROR TypeError: Cannot read property 'lat' of undefined
    at l.i.updateItmCoordinates (main.fe1a303….bundle.js:1)
    at l.i.setMarkerInternal (main.fe1a303….bundle.js:1)
    at l.setMarker (main.fe1a303….bundle.js:1)
    at l.h.moveToResults (main.fe1a303….bundle.js:1)
    at l.h.selectResults (main.fe1a303….bundle.js:1)
    at main.fe1a303….bundle.js:1
    at e.object.i [as _next] (vendor.a85cfb9….bundle.js:1)
    at e.T14+.e.__tryOrUnsub (vendor.a85cfb9….bundle.js:1)
    at e.T14+.e.next (vendor.a85cfb9….bundle.js:1)
    at e.T14+.e._next (vendor.a85cfb9….bundle.js:1)
```

**Comment by SwiftFast** [comment]
Firefox:
```
 TypeError: i.latLng is undefined
Stack trace:
l/i.updateItmCoordinates@http://israelhiking.osm.org.il/main.fe1a3038ad9b1529afc4.bundle.js:1:646352
l/i.setMarkerInternal@http://israelhiking.osm.org.il/main.fe1a3038ad9b1529afc4.bundle.js:1:646131
.jvkb/o</l.prototype.setMarker@http://israelhiking.osm.org.il/main.fe1a3038ad9b1529afc4.bundle.js:1:646691
l/h.moveToResults@http://israelhiking.osm.org.il/main.fe1a3038ad9b1529afc4.bundle.js:1:33083
l/h.selectResults@http://israelhiking.osm.org.il/main.fe1a3038ad9b1529afc4.bundle.js:1:33831
l/<@http://israelhiking.osm.org.il/main.fe1a3038ad9b1529afc4.bundle.js:1:37105
["/oeL"]/</ua</e.prototype.subscribe/i<@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:78872
["T14+"]/u</e.prototype.__tryOrUnsub@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:655721
["T14+"]/u</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:654996
["T14+"]/l</e.prototype._next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:654038
["T14+"]/l</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:653721
.rlar/p</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1542229
["/oeL"]/</ua</e.prototype.emit@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:78321
.bm2B/yt</t.prototype.updateValueAndValidity@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1118207
.bm2B/_t</e.prototype.setValue@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1121599
_/<@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1087300
.Z04r/ka</t.prototype._setValueAndClose@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1039905
.Z04r/ka</t.prototype._subscribeToClosingActions/<@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1039463
["T14+"]/u</e.prototype.__tryOrUnsub@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:655721
["T14+"]/u</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:654996
["T14+"]/l</e.prototype._next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:654038
["T14+"]/l</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:653721
["1eQ7"]/l</e.prototype._emitFinal@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:135664
["1eQ7"]/l</e.prototype._emit@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:135404
["1eQ7"]/l</e.prototype._next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:135140
["T14+"]/l</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:653721
.aCMF/l</e.prototype.notifyNext@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1081102
.kWk1/o</e.prototype._next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1461567
["T14+"]/l</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:653721
.yW9Z/o</e.prototype.notifyNext@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1648966
.kWk1/o</e.prototype._next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1461567
["T14+"]/l</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:653721
.yW9Z/o</e.prototype.notifyNext@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1648966
.kWk1/o</e.prototype._next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1461567
["T14+"]/l</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:653721
["/oeL"]/</ua</e.prototype.subscribe/i<@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:78562
["T14+"]/u</e.prototype.__tryOrUnsub@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:655721
["T14+"]/u</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:654996
["T14+"]/l</e.prototype._next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:654038
["T14+"]/l</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:653721
.rlar/p</e.prototype.next@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1542229
["/oeL"]/</ua</e.prototype.emit@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:78321
.Z04r/Ge</t.prototype._emitSelectionChangeEvent@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:748734
.Z04r/Ge</t.prototype._selectViaInteraction@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:748486
t/<@http://israelhiking.osm.org.il/main.fe1a3038ad9b1529afc4.bundle.js:1:71143
Xn/b@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:39402
xr/<.handleEvent@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:46735
ee@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:16047
Oe/<@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:21664
y/<@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:1389428
.eFQL/</</</f</t.prototype.invokeTask@http://israelhiking.osm.org.il/polyfills.a0a36c25be64dfb2934e.bundle.js:1:36563
["/oeL"]/</ca</t.prototype.forkInnerZoneWithAngularBehavior/this.inner<.onInvokeTask@http://israelhiking.osm.org.il/vendor.a85cfb95ea636746f246.bundle.js:1:81656
.eFQL/</</</f</t.prototype.invokeTask@http://israelhiking.osm.org.il/polyfills.a0a36c25be64dfb2934e.bundle.js:1:36476
.eFQL/</</</u</r.prototype.runTask@http://israelhiking.osm.org.il/polyfills.a0a36c25be64dfb2934e.bundle.js:1:31731
t/this.invoke@http://israelhiking.osm.org.il/polyfills.a0a36c25be64dfb2934e.bundle.js:1:37545
  vendor.a85cfb95ea636746f246.bundle.js:1:6969
```

**Comment by SwiftFast** [comment]
It's worth noting that I recently drastically changed that POI, perhaps it's related.

**Comment by HarelM** [comment]
This is a bug in leaflet:
https://github.com/Leaflet/Leaflet/issues/5499
I'm waiting on @hesher to update the leaflet version in order to fix this as part of #380 
I'm currently marking this as duplicate, if fixing #380 doesn't help please reopen this one.
